en/reference/query-api-reference.html

```diff
@@ -946,6 +947,16 @@ <h3 id="presentation.timing">presentation.timing</h3>
 to the rendered page.
 </p>
 
+<h3 id="presentation.format.tensor">presentation.format.tensors</h3>
```

Comment:
Currently, this is just "format.tensors": https://github.com/vespa-engine/vespa/blob/master/container-search/src/main/java/com/yahoo/search/rendering/JsonRenderer.java#L82

Reply:
Yes, I have a separate PR on that: https://github.com/vespa-engine/vespa/pull/21092
(format.tensors is an alias).